game-core/src/main/java/org/triplea/lobby/common/IModeratorController.java

```diff
@@ -44,14 +42,7 @@ default void banIp(final INode node, final @Nullable Date banExpires) {}
    *
    * @param banExpires {@code null} for a permanent ban.
    */
-  void banMac(INode node, @Nullable Date banExpires);
-
-  /**
-   * Ban the mac.
-   *
-   * @param banExpires {@code null} for a permanent ban.
-   */
-  void banMac(INode node, String hashedMac, @Nullable Date banExpires);
+  void banUser(INode node, @Nullable Instant banExpires);
```

Comment:
I wonder if it would make sense to use simple longs, to avoid using "complex" types that are serialized over the network.